.eslintrc.json

```diff
@@ -8,6 +8,7 @@
   },
   "rules": {
     "eqeqeq":0,
+    //"max-len": ["error", { "code": 80 }],
```

Comment:
Looks good Marius, but you have to enable this eslint rule

Reply:
now its done